src/Collateral.hs

```diff
@@ -125,28 +133,36 @@ mkValidator contractInfo@ContractInfo{..} dat interestPayDate ctx = validate
     checkForTokensDos txo = length ((flattenValue . txOutValue) txo) <= 3
 
     validateReturn :: Bool
-    validateReturn = traceIfFalse "borrower nft is not burnt" validateBorrowerNftBurn &&
-                     traceIfFalse "borrower deadline check fail" checkBorrowerDeadLine &&
-                     traceIfFalse "no correct utxo to interestsc found" validateTxOuts
+    validateReturn =
+      traceIfFalse "borrower nft is not burnt" validateBorrowerNftBurn
+      && traceIfFalse "borrower deadline check fail" checkBorrowerDeadLine
```

Comment:
I find it easier to read with `&&` at the end of the line. Then all `traceIfFalse` are in the same columns